jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -44,13 +44,12 @@ static GradleProjectProperties getForProject(
       Project project, GradleBuildLogger gradleBuildLogger) {
     try {
       return new GradleProjectProperties(
-          project, gradleBuildLogger, GradleSourceFilesConfiguration.getForProject(project));
+          project,
+          gradleBuildLogger,
+          GradleSourceFilesConfiguration.getForProject(project, gradleBuildLogger));
+
     } catch (IOException ex) {
-      throw new GradleException(
-          "Obtaining project build output files failed; make sure you have compiled your project "
-              + "before trying to build the image. (Did you accidentally run \"gradle clean "
-              + "jib\" instead of \"gradle clean compileJava jib\"?)",
-          ex);
+      throw new GradleException("Obtaining project build output files failed", ex);
```

Comment:
It may still be helpful to have a possible reason for this happening in the error message, if it's still likely that this could even happen.

Reply:
That will actually just be an `IOException` that happens when trying to list directories, so it's not a likely error anymore.